docs/setup.md

```diff
@@ -22,7 +22,11 @@ git config --global core.quotepath off
 The library was built using vscode. VScode Jedi didn't display tamil function names in outline window and intellisense. To fix this switch to language server instead of Jedi (set "python.jediEnabled": false in your settings.json). I have raised a bug with Vscode  
 
  [Function names with unicode (indic characters) are not displayed in Outline window #6454](https://github.com/microsoft/vscode-python/issues/6454)
- 
+
+### python package installation
+இந்த நிரல் தொகுப்பை நிறுவவதற்கு இப்படி pip செயலியில் கட்டளை இடவேண்டும்.
+$ python3 -m pip install setup.py  --user

```

Comment:
இந்த நிரல் தொகுப்பை நிறுவவதற்கு இப்படி pip செயலியில் கட்டளை இடவேண்டும். I myself took some time to understand this text. I refrained from writing in Tamil where it is not required or add value. If you think that way request you to change this to English.

Reply:
இது தாங்க தமிழ் கணிமையில் உள்ள மொழி. எனக்கு மாற்றுவதில் உடன்பாடு கிடையாது - ஆங்கிலத்திலும் எழுதிடலாம் என்றால் சற்று குழப்பம் இல்லாமல் இருக்கும் என்று நம்புகிறேன். சரியா ?

Reply:
Sure we can do in English and Tamil. English will help no Tamil developers to understand the framework